### DISCUSSION TITLE
anyone else problems with this site loading?

### DISCUSSION THREAD

**Discussion by thibaultmol** [Divested-Mobile/Mull-Fenix#266] [Q&A]
https://randombutmemorable.simplecast.com/episodes/safe-search-biscuit-record

I only see the cookie prompt but the page is blank. Same in private mode. Firefox does work though

**Comment by SkewedZeppelin** [comment]
https://divestos.org/pages/broken#mull
> Mull has stripped referrers. This often breaks loading of images on websites with hotlink protection. Navigate to about:config and change network.http.referer.XOriginPolicy from 2 to 1, this is however a privacy risk.

In this case changing it to 0 appears required.

Since it is a global option, maybe consider another browser like Fennec F-Droid or a dedicated app like AntennaPod: https://f-droid.org/packages/de.danoeh.antennapod/

**Reply by thibaultmol** [reply]
- I have a dedicated podcast app, just wanted to know more about the episode before switching to my podcast app
- I actually switched from Fennec when it wasn't updating a couple weeks ago

is there something I could tell the website creator about something that should be fixed about it?

**Reply by SkewedZeppelin** [reply]
it is likely wholly intentional on their end

**Reply by thibaultmol** [reply]
> it is likely wholly intentional on their end

ok, but what exactly do i need to tell them to fix?

**Reply by SkewedZeppelin** [reply]
XOriginPolicy 0 sends full URL in referrer field
if they were to fix this on their end they need to stop checking that the referrer for requests matches the page loading
